app/models/submission_collector.rb

```diff
@@ -238,20 +238,9 @@ def manually_collect_submission(grouping, rev_num,
     self.safely_stop_child_exited = true
     self.save
 
-    #Let the child process handle conversion, as things go wrong when both
-    #parent and child do this.
-    start_collection_process do
```

Comment:
IS this still necessary? (from line 249)
    #setting is_collected here will prevent an sqlite lockout error when pdfs
    #aren't supported
      grouping.is_collected = true
      grouping.save